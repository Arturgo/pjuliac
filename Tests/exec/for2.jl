
# les bornes de for ne sont évaluées qu'une seule fois

function f(x)
    print(x, ".")
    x
end

for n = f(1) : f(10)
    print(n)
end
println()
