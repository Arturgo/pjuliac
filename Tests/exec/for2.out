1.10.12345678910
